src/stryker-cli.ts

```diff
@@ -2,17 +2,20 @@ import * as path from 'path';
 import * as chalk from 'chalk';
 import * as inquirer from 'inquirer';
 import * as child from 'child_process';
-import RequireWrapper from './RequireWrapper';
+import NodeWrapper from './NodeWrapper';
+import * as resolve from 'resolve';
 
-export function run(log = console.log): Promise<void> {
+const installCommand = 'npm install --save-dev stryker stryker-api';
```

Comment:
I'm not sure what the standard in TS is, but in Java it is common practice to name constants in uppercase. e.g. `INSTALL_COMMAND`. Perhaps this could make the intent of this value more clear.

Reply:
Yeah, i've seen that for some typescript/javascript projects as well. It's not a general rule. The Angular team for example uses it (if i'm not mistaken), but the TypeScript team does not. We don't use it either. If you want, we can do it, i'm not strongly against it or anything.

Nothing about it in this styleguide:

https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md

Reply:
If you don't already use it in the main project, then I wouldn't start using it here. Consistency is often the best option. Let's keep it how it is.